docs/configuration/backends/kubernetes.md

```diff
@@ -50,6 +50,16 @@ See also [Kubernetes user guide](/user-guide/kubernetes).
 #
 # labelselector = "A and not B"
 
+# Value of `kubernetes.io/ingress.class` annotation that identifies Ingress objects to be processed.
+# Note that Ingress objects with an empty or nonexistent `kubernetes.io/ingress.class` value
+# will also be processed.
+# Note that if `kubernetes.io/ingress.class` annotation  is not empty, only annotations with the prefix `traefik` will be processed.
+#
+# Optional
+# Default: empty (process all Ingresses)
+#
+# ingressClass = "traefik-custom-proxy"
```

Comment:
I'd use the example value `traefik-internal` -- many users actually come with this specific use case.